src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -170,9 +159,15 @@ def __init__(
 
         self._enable_widgets = self._pl._enable_widgets
 
-        self._hover_picker = vtkPointPicker() if self. _allow_hovering else None
-        self._hover_widget = vtkHoverWidget() if self. _allow_hovering else None
-        self._added_hover_labels = []
+        self._hover_picker = vtkPointPicker() if self._allow_hovering else None
+        self._hover_widget = vtkHoverWidget() if self._allow_hovering else None
+
+        if custom_picker is None:
+            self._custom_picker = Picker(self, self._plot_picked_names)
+        elif issubclass(custom_picker, AbstractPicker):
+            self._custom_picker = custom_picker(self, self._plot_picked_names)
```

Comment:
Due to the way you are passing inputs to the custom picker's ctor, I think it'd be best to to modify it's init to force the same ctor as the normal Picker. Or accept an input called ``custom_picker_args`` in the PyVista plotter that accepts a dictionary (with kwargs inside)... and then you just do:

```py
self._custom_picker = custom_picker(self, **custom_picker_args)

Reply:
Thinking about it, why not just remove the init from the abstract class and let users fully define their init? I think it's almost the same as having an abstract init with kwargs